generator/AWSPSGeneratorLib/HelpMaterials/Examples/ELB2/Remove-ELB2ListenerCertificate.xml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$certificate = New-Object Amazon.ElasticLoadBalancingV2.Model.Certificate
+$certificate.CertificateArn = 'arn:aws:acm:us-east-1:123456789012:certificate/19478bd5-491d-47d4-b1d7-5217feba1d97'
+
+Remove-ELB2ListenerCertificate -Certificate $certificate -ListenerArn 'arn:aws:elasticloadbalancing:us-east-1:123456789012:listener/app/test-alb/3651b4394dd9a24f/3873f123b98f7618' -Region us-east-1
```

Comment:
Can be simplified to `-Certificate @{CertificateArn = 'arn:aws:acm:us-east-1:123456789012:certificate/19478bd5-491d-47d4-b1d7-5217feba1d97'}`